templates/secret.yaml

```diff
@@ -38,7 +38,11 @@ data:
   
   {{- if .Values.proxy.username }}
   proxyUsername: {{ .Values.proxy.username | b64enc | quote }}
+  {{- else }}
+  proxyUsername: ""
   {{- end }}
   {{- if .Values.proxy.password }}
   proxyPassword: {{ .Values.proxy.password | b64enc | quote }}
+  {{- else }}
+  proxyPassword: ""
```

Comment:
Rather than if-blocks, would a ` | default ""` work here?

e.g: `proxyPassword: {{ .Values.proxy.password | default "" | b64enc | quote }}`

If so, same could be done with proxyUsername and make this changeset super tiny!

Reply:
Yes, i agree.
have pushed the new changes using `default` function.

Reply:
Hi,
Yeah, i have removed the [If-blocks](https://github.com/twuni/docker-registry.helm/pull/25/files) since that returns false when `.Values.proxy.username = "`.